README.md

```diff
@@ -684,6 +690,13 @@ const evenNumbers = numbers.filter(function(n) {
 console.log(evenNumbers); // [0, 2, 4, 6]
 ```
 
+In ES6
```

Comment:
Same

Reply:
I have made the necessary changes as mentioned above